.github/workflows/R2S-OpenWrt-Without-Docker.yml

```diff
@@ -113,7 +113,7 @@ jobs:
       id: create_release
       uses: actions/create-release@v1
       env:
-        GITHUB_TOKEN: ${{ secrets.workflow_token }}
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
```

Comment:
Do you know "GITHUB_TOKEN" will be expired in an hour?
Or do you consider the task can be done in an hour?

Reply:
I don't think so. Check this link, the compilation takes more than 3 hours and GITHUB_TOKEN is still valid. https://github.com/02015678/NanoPi-R2S/actions/runs/220940943/workflow

Reply:
You can omit the change of this file. The drawback of using GITHUB_TOKEN is that, the release is uploaded under the name of Github Action instead of yourself.

Reply:
https://github.community/t/github-token-lifetime/16837
Maybe GitHub made some changes recently.
According to the latest document, now GITHUB_TOKEN seems will expire only if the job is done.
However, we have set the token already, so it's pointless to replace.